src/containers/JudgingPanel.js

```diff
@@ -46,16 +47,67 @@ class Project {
   }
 }
 
+const getGradedProjects = async () => {
+  const projectDocs = await projectsRef.get()
+  const projectData = projectDocs.docs.map(projectDoc => {
+    const project = projectDoc.data()
+    if (project.grades) {
+      project.countGraded = Object.values(project.grades).length
```

Comment:
thoughts on doing `Object.values()` in `formatProject`? I'm doing this cast too :(

Reply:
like in general or here? I am not using `formatProject` here, should I be?